runtime/include/blueoil_image.hpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2018 Leapmind Inc. */
+#ifndef RUNTIME_INCLUDE_BLUEOIL_IMAGE_HPP_
+#define RUNTIME_INCLUDE_BLUEOIL_IMAGE_HPP_
+
+#include "blueoil.hpp"
+
+namespace blueoil {
+namespace image {
+
+float *Tensor_at(Tensor &tensor, const int x, const int y); // return RGB
+Tensor Tensor_CHW_to_HWC(Tensor &tensor);
+Tensor Tensor_HWC_to_CHW(Tensor &tensor);
+
+enum ResizeFilter {
+		   RESIZE_FILTER_NEAREST_NEIGHBOR = 0,
+		   RESIZE_FILTER_BI_LINEAR
+};
+
+Tensor Resize(const Tensor& image, const int width, const int height,
+	      const enum ResizeFilter filter);
```

Comment:
This function seems to be just a part of implementation of
```cpp
Tensor Resize(const Tensor& image, const std::pair<int, int>& size) {
```
which is in another file...

What are the benefits of splitting these functions across two files, when we could have just one file with only `￼Tensor Resize(const Tensor& image, const std::pair<int, int>& size)` exported as public interface instead...

Reply:
I think that it's better to implement image processing like Resize in a general-purpose, and into blueold_image.cpp. After this we will also want to process grayscale, cropping, flipping, equalize etc.
In addition, it's better to gathering the acceptance of processing corresponding to the YAML rule in blueoil_data_processor.cpp.